deployment/cdk/opensearch-service-migration/lib/migration-assistance-stack.ts

```diff
@@ -179,8 +156,25 @@ export class MigrationAssistanceStack extends Stack {
             instanceType: InstanceType.of(InstanceClass.T2, InstanceSize.MICRO),
             machineImage: MachineImage.latestAmazonLinux(),
             securityGroup: oinoSecurityGroup,
-            // Manually created for now, to be automated soon
-            keyName: "es-node-key"
+            // Manually created for now, to be automated in future
+            //keyName: "es-node-key"
+        });
+
+        // This is a temporary fragile piece to help with importing values from CDK to Copilot. It assumes the provided VPC has two public subnets and
+        // does not currently provide the MSK broker endpoints as a future Custom Resource is needed to accomplish this
```

Comment:
Do you have a task to clean this up.  It would be worth documenting why we're not deploying the bitnami container - what issues you hit, etc.  Having a ticket to reference would be good too.

Reply:
Will create a task and add here. I'm weary to put any discussion here as this seems highly subject to change as we decide how we want to proceed